en-US/snippets/2017/mar2017_b.lang

```diff
@@ -6,115 +6,148 @@
+# Campaign: desktop snippets
 ;Send yourself a download link for a Web without limits.
 Send yourself a download link for a Web without limits.
 
 
 ## MAX_LENGTH: 180 characters
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 # "Citation needed" is meant to be humorous.
 ;Fun fact: Firefox users are the smartest, funniest, best-looking people on the Web. [Citation needed]
 Fun fact: Firefox users are the smartest, funniest, best-looking people on the Web. [Citation needed]
 
 
 ## MAX_LENGTH: 180 characters
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 ;Need a quick recharge? Power up with a power nap. Geniuses like Dali and Einstein loved sneaking in some extra ZZZs.
 Need a quick recharge? Power up with a power nap. Geniuses like Dali and Einstein loved sneaking in some extra ZZZs.
 
 
 ## MAX_LENGTH: 180 characters
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 # Context: A mic dropper is a brash, talented person who isn't afraid of a little controversy. Mic dropping is the act of dropping a microphone, a gesture intended to highlight the end of an amazing or shocking performance or truthful statement. If it's time consuming to find a witty quote from a famous author or poet of your country who is no longer with us, you can simply translate the following snippet.
 ;Nonconformist Hall of Fame: RIP original mic dropper Oscar Wilde. To wit: “Always forgive your enemies; nothing annoys them so much.”
 Nonconformist Hall of Fame: RIP original mic dropper Oscar Wilde. To wit: “Always forgive your enemies; nothing annoys them so much.”
 
 
 ## MAX_LENGTH: 180 characters
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 # Link: https://support.mozilla.org/kb/forget-button-quickly-delete-your-browsing-history
 ;Unlike your attic, Firefox is easy to clean. <a>Just click the Forget Button and presto!</a> No more browser history.
 Unlike your attic, Firefox is easy to clean. <a>Just click the Forget Button and presto!</a> No more browser history.
 
 
 ## MAX_LENGTH: 180 characters
+# LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
+# Campaign: desktop snippets
 # Link: https://addons.mozilla.org/firefox/themes
 ;Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
 Your browser is ready for a makeover. Don’t worry, you’ve got tons of options. <a>Check them out here.</a>
 
 
-# Appears on Mobile Activation screen; Refer to the screenshot.
+# Appears on Mobile Activation screen; For context, refer to https://bug1336204.bmoattachments.org/attachment.cgi?id=8833047
+# This is a button. Be mindful of the character limit.
```

Comment:
I would say "keep the translation as short as possible", given that we don't tell them what the limit is.